code/src/components/SendButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const SendButton = (message) => {
+  return (
+    <button
+      type="submit"
+      className="sendButton"
+      disabled={message.length < 6 || message.length > 140}>
+      ❤️Send a Happy Thought❤️
+    </button>
+  )
+}
```

Comment:
I wonder if this disabled is the source of your page breaking when you enter <5 or more than 140characters. what about something like this so its an if else statement.         
```
export const SendButton = ({ message }) => {
  const [isDisabled, setIsDisabled] = useState(true);

  const handleDisabled = (message) => {
    if (message.length < 6 || message.length > 140) {
      setIsDisabled(true);
    } else {
      setIsDisabled(false);
    }
  };

  handleDisabled(message);

  return (
    <button
      type="submit"
      className="sendButton"
      disabled={isDisabled}
    >
      ❤️Send a Happy Thought❤️
    </button>
  );
};
```